xarray/core/options.py

```diff
@@ -6,6 +6,7 @@
 OPTIONS = {
     'display_width': 80,
     'arithmetic_join': 'inner',
+    'engine': None,
```

Comment:
Should this have the current default rather than `None`?

Reply:
I did not want to alter the current behavior. My understanding is that unless a user specifies an engine it is effectively `None` which then invokes a `_get_default_engine()` call to figure out the most appropriate one based on the file path.